public/app/plugins/panel/logs/LogsPanel.test.tsx

```diff
@@ -306,12 +368,13 @@ describe('LogsPanel', () => {
       rerender(<LogsPanel {...props} />);
 
       expect(await screen.findByRole('row')).toBeInTheDocument();
-      expect(styles.getLogRowStyles).toHaveBeenCalledTimes(3);
```

Comment:
FYI, after much debugging I realized that with the way the mock props were used before, `props.request` was undefined, causing less "stuff" to happen within the component. You can easily verify this in main or in [this branch](https://github.com/grafana/grafana/tree/matyax%2Fdoesn-t-lie-or-maybe).